internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr.go

```diff
@@ -239,6 +244,20 @@ func (k *KafkaManager) calculateCapacityByRegionAndInstanceTypeForManualClusters
 		return totalUsed, instanceTypeUsed
 	}
 
+	// enterprise data plane clusters
```

Comment:
Is an enterprise cluster considered a "manual" cluster? Maybe this is not the most appropriate place where to put this logic?

Reply:
I'll move it out this function